engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/EnsureCompiledJob.scala

```diff
@@ -347,37 +349,6 @@ final class EnsureCompiledJob(
     )
   }
 
-  type IDMap = Seq[(org.enso.data.Span, java.util.UUID)]
-  private def splitMeta(code: String): (String, IDMap, io.circe.Json) = {
-    import io.circe.Json
```

Comment:
Wouldn't this, by a chance, allow us to remove circe from some of our project dependencies?

Reply:
Nope. We use circe for parsing configuration files as well as parsing requests.